### PR TITLE
Add a new section for runtime download components.

### LINKED ISSUE #22203: .NET Core runtime download components in introduction.md file.
**Issue description**

It is clear from the introduction.md documentation file that what is included in SDK download but not clear what is included in runtime download. So it needs to be mentioned in the documentation.

**Target framework**

- [x] .NET Core
- [ ] .NET Framework
- [ ] .NET Standard

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3ad87897-30cc-e639-b91a-b3800269f7fe
* Version Independent ID: 9d3015b4-cba1-0956-9ec7-4154dbb74d93
* Content: [.NET introduction and overview](https://docs.microsoft.com/en-us/dotnet/core/introduction)
* Content Source: [docs/core/introduction.md](https://github.com/dotnet/docs/blob/master/docs/core/introduction.md)
* Product: **dotnet-fundamentals**
* GitHub Login: @tdykstra
* Microsoft Alias: **dotnetcontent**

### PR DESCRIPTION
**Pull request by Arnab-Developer**
## Summary

It is clear from the introduction.md documentation file that what is included in SDK download but not clear what is included in runtime download. So I have made a new section and add some points for runtime download components in introduction.md file.

Fixes #22203

### CODE REVIEW

File: docs/core/introduction.md [hunk 1]
```diff
@@ -139,6 +139,13 @@ The SDK download includes the following components:
 * The ASP.NET Core runtime. Provides basic services for internet-connected apps, such as web apps, IoT apps, and mobile backends.
 * The desktop runtime. Provides basic services for Windows desktop apps, including Windows Forms and WPF.
 
+The runtime download includes the following components:
+
+* The desktop or ASP.NET Core runtime.
```
Comment (tdykstra) [inline docs/core/introduction.md:144]:
```suggestion
* Optionally, the desktop or ASP.NET Core runtime.
```

**Review by tdykstra — APPROVED** [review]
@Arnab-Developer Thanks for submitting this PR! I made one suggestion.

File: docs/core/introduction.md (hunk 1 above)
Comment (Arnab-Developer) [reply docs/core/introduction.md:144]: As a part of .NET runtime download, we need to download either ASP.NET Core runtime or desktop runtime. The first point is mandatory. So why we should make the first point as optional?

https://dotnet.microsoft.com/download/dotnet/current/runtime

Comment (tdykstra) [reply docs/core/introduction.md:144]: See this page:  https://dotnet.microsoft.com/download/dotnet/5.0

Comment (Arnab-Developer) [reply docs/core/introduction.md]: Thanks for the information. I have made the suggested changes.

**Pull request merged**